cleanlab/datalab/datalab.py

```diff
@@ -89,6 +96,7 @@ def __init__(
         self,
         data: "DatasetLike",
         label_name: Optional[str] = None,
+        image_key: Optional[str] = None,
```

Comment:
The documentation for this arg is totally missing...

Here's a suggested docstring:

```
image_key : str
    Optional key that can be specified for image datasets to point to the field containing the actual images themselves.
    If specified, additional image-specific issue types can be detected in the dataset.
    See the CleanVision package documentation for descriptions of these image-specific issue types.
```